compose/src/main/java/com/maplibre/compose/symbols/Polyline.kt

```diff
@@ -24,16 +29,35 @@ import com.maplibre.compose.ramani.PolyLineNode
 @MapLibreComposable
 fun Polyline(
     points: List<LatLng>,
-    color: String,
+    color: String? = null,
     lineWidth: Float,
     zIndex: Int = 0,
     isDraggable: Boolean = false,
     isDashed: Boolean = false,
     lineCap: String = Property.LINE_CAP_ROUND,
-    lineJoin: String = Property.LINE_JOIN_ROUND
+    lineJoin: String = Property.LINE_JOIN_ROUND,
+    linePatternId: Int? = null
 ) {
+    val context = LocalContext.current
     val mapApplier = currentComposer.applier as MapApplier
 
+    linePatternId?.let {
+        val id = it.toString()
+        if (mapApplier.style.getImage(id) == null) {
+            val vectorDrawable = context.getDrawable(it) as? VectorDrawable
+            if (vectorDrawable == null) {
+                mapApplier.style.addImage(
+                    id,
+                    ImageBitmap.imageResource(it).asAndroidBitmap()
+                )
+            } else {
+                vectorDrawable.let { drawable ->
+                    mapApplier.style.addImage(id, drawable)
+                }
+            }
+        }
+    }
```

Comment:
This is basically copied from the `Symbol` implementation. Should we extract? I kind of prefer the impl here since it uses the refined inner variable instead of the nullable outer variable, but open to feedback.

Reply:
Let's pull out a helper

Reply:
And add ktfmtFormat